src/main/java/com/github/thedeathlycow/frostiful/mixins/compat/colorfulhearts/absent/ColdHeartOverlay.java

```diff
@@ -13,8 +15,10 @@
 @Mixin(InGameHud.class)
 abstract class ColdHeartOverlay {
 
+    @Unique
     private final int[] heartYPositions = new int[FrozenHeartsOverlay.MAX_COLD_HEARTS];
-    private final int[] heartXPositions = new int[FrozenHeartsOverlay.MAX_COLD_HEARTS];;
+    @Unique
+    private final int[] heartXPositions = new int[FrozenHeartsOverlay.MAX_COLD_HEARTS];
```

Comment:
Now that there are two mods that are dealing with this, duplication of these lines is starting to become a problem. I think the arrays should be moved into the `FrozenHeartsOverlay` singleton and be updated from the mixins here. Would also remove these fields from the mixin classes, which is probably for the best.

Reply:
Easy to do, shall be done